components/seldon-request-logger/app/log_helper.py

```diff
@@ -85,6 +85,11 @@ def parse_message_type(type_header):
         or type_header == "org.kubeflow.serving.inference.response"
     ):
         return "response"
+    if (
```

Comment:
We could probably build a dictionary having mapping: header -> message type

Reply:
We can do on refactor as this whole request logger will need refactor